pkg/services/sqlstore/mockstore/mockstore.go

```diff
@@ -44,7 +44,8 @@ type SQLStoreMock struct {
 	ExpectedUserStars              map[int64]bool
 	ExpectedLoginAttempts          int64
 
-	ExpectedError error
+	ExpectedError            error
+	ExpectedSetUsingOrgError error
```

Comment:
Curious, why is this change needed?

Reply:
One terrible unit tests that expects storage to succeed on all lookups, but fail on SetUsingOrg 🤷 
Better leave this until the store split than figure out how to fix that test.

Reply:
The relationship between OrgRedirect and ContextHandler should be revisited, currently they rely on too many implementation details which leads to fragile tests and complicates the refactoring.